corehq/apps/user_importer/importer.py

```diff
@@ -485,9 +485,19 @@ def create_or_update_commcare_users_and_groups(upload_domain, user_specs, upload
             update_progress(current)
             current += 1
 
+        status_row = {}
         username = row.get('username')
         domain = row.get('domain') or upload_domain
-        username = normalize_username(str(username), domain) if username else None
+        try:
+            username = normalize_username(str(username), domain) if username else None
```

Comment:
I see that `normalize_username` is deprecated, it is advised to use `generate_mobile_username` instead.
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/users/util.py#L85-L108

Reply:
Yeah so there are two reasons why I didn't switch over to `generate_mobile_username` here.